scripts/beachball/index.ts

```diff
@@ -1,13 +1,33 @@
 import { BeachballConfig } from 'beachball';
+import { getAllPackageInfo } from '../monorepo/index';
 import { renderHeader, renderEntry } from './customRenderers';
 
+const allPackageInfo = getAllPackageInfo();
+const fluentConvergedPackagePaths = Object.values(allPackageInfo)
+  .map(packageInfo => {
+    if (packageInfo.packageJson.version.startsWith('9.') && !packageInfo.packageJson.private) {
+      return packageInfo.packagePath;
+    }
+
+    return false;
+  })
+  .filter(Boolean) as string[];
```

Comment:
Why do a map before a filter instead of filtering and then getting only the `packagePath`?

Reply:
I just want an array of paths to packages which are converged (i.e. v9). Not all packages in the repo satisfy that constraint.

`[@fluentui/react-button, @fluentui/react]` -> map -> `['packages/react-button', false]` -> filter -> `['packages/react-button']`

If there's a better way to do this then, I'd be interested to know